### PR TITLE
[GEOT-6145]: ForceComponentColorModel noAlpha

### PR DESCRIPTION
**Pull request by dromagnoli**

### CODE REVIEW

File: modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java [hunk 1]
```diff
@@ -1747,20 +1747,24 @@ public final ImageWorker forceComponentColorModel() {
...
      * @return this {@link ImageWorker}.
      * @see FormatDescriptor
      */
     public final ImageWorker forceComponentColorModel(
-            boolean checkTransparent, boolean optimizeGray) {
+            boolean checkTransparent, boolean optimizeGray, boolean omitAlpha) {
         final ColorModel cm = image.getColorModel();
         if (cm instanceof ComponentColorModel) {
-            // Already an component color model - nothing to do.
+            // Already a component color model - nothing to do.
```
Comment (aaime) [inline modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java]: What if it was a component color model with alpha and omitAlpha was false? The javadoc could be made more explicit, and state that the color model is already component, then the alpha is preserved. Also, "omitAlpha" could be renamed to "omitAlphaOnExpand"

Comment (dromagnoli) [reply modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java]: Thx. Going to update it, accordingly.

Comment (dromagnoli) [reply modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java]: Done

**Comment by aaime** [comment]
Thank you, good to go imho.

**Pull request merged**